recipes/aws-cdi-sdk/all/conanfile.py

```diff
@@ -24,19 +29,17 @@ def _source_subfolder(self):
     _autotools = None
     _cmake = None
 
-    def configure(self):
-        self.options["aws-sdk-cpp"].monitoring = True
-        self.options["aws-sdk-cpp"].shared = True
-        self.options["aws-libfabric"].shared = True
-
 
     def validate(self):
         if self.settings.os != "Linux":
-            raise ConanInvalidConfiguration("The aws-cdi-sdk con only be built on Linux.")
+            raise ConanInvalidConfiguration("This recipe currently only supports Linux. Feel free to contribute other platforms!")
         elif (self.settings.compiler == "gcc"
                 and tools.Version(self.settings.compiler.version) < "6.0"):
             raise ConanInvalidConfiguration("""Doesn't support gcc5 / shared.
             See https://github.com/conan-io/conan-center-index/pull/4401#issuecomment-802631744""")
+        if not self.options["aws-libfabric"].shared or self.options["aws-sdk-cpp"].shared:
```

Comment:
I am not sure how python handles this but maybe this is easier to read?

```suggestion
        if not self.options["aws-libfabric"].shared or not self.options["aws-sdk-cpp"].shared:
```

Reply:
good catch